package.json

```diff
@@ -107,6 +107,7 @@
     "mocha": "4.1.0",
     "module-alias": "2.2.0",
     "monaco-editor": "0.15.6",
+    "mutationobserver-shim": "0.3.3",
```

Comment:
what is this needed for here?

Reply:
https://react-hook-form.com/faqs/#TestingfailedduetoMutationObserver

Reply:
I do think it isn't needed when doing a shallow render, but rather when you mount it. And I scrapped those tests because of the `act` issues etc.

Reply:
I see now. Haven't noticed jest setup file update, sorry